src/Plugin/tableview/Options.ts

```diff
@@ -101,7 +101,18 @@ export default class Options {
 			 * @example
 			 *   legendToggleUpdate: false
 			 */
-			updateOnToggle: true
+			updateOnToggle: true,
+
+			/**
+			 * Shows null values as X in the table
+			 * @name showNulls
+			 * @memberof plugin-tableview
+			 * @type {boolean}
+			 * @default false
+			 * @example
+			 *   showNulls: false
+			 */
+			showNulls: false
```

Comment:
IMO isn't really necessary to add new option here. when the value is null, just showing a "blank" will be okay.